src/plugins/wizard/public/visualizations/bar_chart/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { VisualizationTypeOptions } from '../../services/type_service/visualization_type';
+
+export const createBarChartConfig = (): VisualizationTypeOptions => {
+  return {
+    name: 'bar_chart',
+    title: 'Bar Chart',
+    icon: 'visBarVertical',
+    description: 'This is a bar chart',
```

Comment:
Is it important to localize these?

Reply:
Yep. Once I finalize the API here, ill update the strings in #958